distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1552,6 +1571,130 @@ public void testIngestPlugin() throws Exception {
+        networkRule2Properties.put("expectedValue", clientScheme);
+        networkRule2Properties.put("newAttributes", attributeAdjustmentsForRule2);
+
+        getServiceManager().createManagedService(factoryPid, networkRule1Properties);
+        getServiceManager().createManagedService(factoryPid, networkRule2Properties);
+
+        getServiceManager().waitForRequiredBundles(symbolicName);
+
+        // ** INGEST **
+        // ** Working with two metacard xml files and one txt file to serve as a tika product **
+        String simpleProductId = ingestMetacardAndGetId("simple-product.txt", MediaType.TEXT_PLAIN);
+        String card1Id = ingestMetacardAndGetId("modified-metacard-1.xml", MediaType.TEXT_XML);
+        String card2Id = ingestMetacardAndGetId("modified-metacard-2.xml", MediaType.TEXT_XML);
+
+        // ** VALIDATION **
+        // ** Note only the first validation gets security since it's parsed by tika **
+        startValidateBody(simpleProductId).assertThat()
+                .body(hasNewDescription)
+                .body(hasNewMetacardTags)
+                .body(hasNewSecurityAccessGroups);
+        startValidateBody(card1Id).assertThat()
+                .body(hasInitialDescription)
+                .body(hasNewMetacardTags);
+        startValidateBody(card2Id).assertThat()
+                .body(hasNewDescription)
+                .body(hasInitialMetacardTags);
+
+        // ** CLEAN UP **
+        // ** Don't let the conditions spill-over and impact other tests **
+        getServiceManager().stopFeature(true, "catalog-metacardingest-network");
+
+    }
+
+    private ValidatableResponse startValidateBody(String metacardId) {
+        final String metacardUrlFormat =
+                SERVICE_ROOT.toString() + "/catalog/sources/ddf.distribution/%s";
+        return given().expect()
+                .log()
+                .all()
+                .statusCode(HttpStatus.SC_OK)
+                .when()
+                .get(URI.create(format(metacardUrlFormat, metacardId)))
+                .then();
+    }
+
+    private String ingestMetacardAndGetId(String fileName, String contentType) {
+        return given().body(getFileContent(fileName))
+                .header(HttpHeaders.CONTENT_TYPE, contentType)
+                .expect()
+                .log()
+                .all()
+                .statusCode(HttpStatus.SC_CREATED)
+                .when()
+                .post(REST_PATH.getUrl())
+                .getHeader("id");
+    }
+
+    private Matcher<Node> hasMetacardAttributeXPath(String attributeName,
+            String... attributeValues) {
+        final String basePathFormat = "/metacard/string[@name=\"%s\"][value=\"%s\"";
+        final String additionalValueFormat = " and value=\"%s\"";
+
+        String xPath = format(basePathFormat, attributeName, attributeValues[0]);
+        for (int i = 1; i < attributeValues.length; i++) {
+            xPath += format(additionalValueFormat, attributeValues[i]);
```

Comment:
⭐️ Using both string concatenation _and_ format at the same time seems a little odd and inefficient. We should use one or the other, or simply use a `StringBuilder`.